examples/src/main/java/io/opencensus/examples/zpages/ZPagesTester.java

```diff
@@ -105,4 +105,4 @@ public static void main(String[] args) throws Exception {
     ZPageHandlers.startHttpServerAndRegisterAll(8080);
     recordExampleData();
   }
-}
+}
```

Comment:
I think the style error is from here - no new line at EOF. Could you revert this change?